README.md

```diff
@@ -1,7 +1,89 @@
 # Northcoders News API
 
+## Hosted Version
+
+The hosted version of this repository can be found at: https://news-ynvj.onrender.com/
+
+## Project Summary
+
+This project is an API for the purpose of accessing application data programmatically.
+
+It mimics the building of a real world backend service.
+
+## Repo set up
+
+### Cloning this repo
+
+This repo can be cloned by running the following command in the CLI:
+
+```
+git clone https://github.com/cCody34/nc-news.git
+```
+
+### npm init -y
+
+Initialise the project by running the following command in the CLI:
+
+```
+npm init -y
+```
+
+### Installing dependencies
+
+The dependencies dotenv, express and pg should be installed using the following commands in the CLI:
+
+```
+npm i dotenv
+npm i express
+npm i pg
+```
+
```

Comment:
These are all listed in the dependencies of your project (that's partially the package.lock is for) - you only need to run `npm i` and these will all be installed without having to name them one by one